### PR TITLE
Renovation: memoized callback ES6 refactoring 

### PR DESCRIPTION
**Pull request by LazyLahtak**
Refactor Memoized Callback into ES6 class

### CODE REVIEW

File: js/core/memorized_callbacks.js [hunk 1]
```diff
@@ -1,27 +1,23 @@
-const each = require('../core/utils/iterator').each;
-const Callbacks = require('./utils/callbacks');
+import { each } from '../core/utils/iterator';
+import Callbacks from './utils/callbacks';
 
-const MemorizedCallbacks = function() {
+export default class MemorizedCallbacks {
+    constructor() {
+        this.memory = [];
+        this.callbacks = Callbacks();
```
Comment (MaximKudriavtsev) [inline js/core/memorized_callbacks.js:7]: Should we rewrite `Callbacks` to `class`?

File: js/events/core/event_registrator_callbacks.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
-const MemorizedCallbacks = require('../../core/memorized_callbacks');
+import MemorizedCallbacks from '../../core/memorized_callbacks';
 
 module.exports = new MemorizedCallbacks();
```
Comment (MaximKudriavtsev) [inline js/events/core/event_registrator_callbacks.js]: Same with export

File: js/core/component_registrator_callbacks.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
-const MemorizedCallbacks = require('./memorized_callbacks');
+import MemorizedCallbacks from './memorized_callbacks';
 
 module.exports = new MemorizedCallbacks();
```
Comment (MaximKudriavtsev) [inline js/core/component_registrator_callbacks.js]: Same with export

File: js/core/memorized_callbacks.js (hunk 1 above)
Comment (LazyLahtak) [reply js/core/memorized_callbacks.js:7]: I think yes, but in other PR.

File: js/events/core/event_registrator_callbacks.js (hunk 2 above)
Comment (LazyLahtak) [reply js/events/core/event_registrator_callbacks.js]: Done

File: js/core/component_registrator_callbacks.js (hunk 3 above)
Comment (LazyLahtak) [reply js/core/component_registrator_callbacks.js]: Done

**Review by MaximKudriavtsev — APPROVED** [review]

**Pull request merged**